src/execution_plan/execution_plan.h

```diff
@@ -28,6 +28,7 @@ typedef struct {
 	QueryGraph *query_graph;           // QueryGraph representing all graph entities in this segment.
 	FT_FilterNode *filter_tree;        // FilterTree containing filters to be applied to this segment.
 	AR_ExpNode **projections;          // Expressions to be constructed for a WITH or RETURN clause.
+	AR_ExpNode **order_expressions;    // Expressions to be constructed for an ORDER clause.
```

Comment:
Why do we need this to be part of the struct?

Reply:
Hm, I should be able to remove this.